src/Data/Mailcap.hs

```diff
@@ -43,9 +43,8 @@ offset = AT.Parser $ \t pos more _lose suc -> suc t pos more (AT.fromPos pos)
 
 type MailcapFile = [MailcapLine]
 
-data MailcapLine
-  = Comment B.ByteString
-  | MailcapEntry Entry
+newtype MailcapLine
+  = MailcapEntry Entry
```

Comment:
The `MailcapLine` type now serves no useful purpose (AFAICT).  We could have just:

```haskell
type MailcapFile = [Entry]
```

and keep it simple :)

(sorry to be a nitpick)

Reply:
Ah nah.. that's alright. Fair point. Should have made the mental connection...

Reply:
Ok should be changed :)